packages/react/src/primitives/types/style.ts

```diff
@@ -17,7 +18,7 @@ export type ResponsiveStyle<PropertyType> =
   | PropertyType[]
   | ResponsiveObject<PropertyType>;
 
-export interface BaseStyleProps extends GridItemStyleProps {
+export interface BaseStyleProps extends FlexItemStyleProps, GridItemStyleProps {
   alignSelf?: ResponsiveStyle<Property.AlignSelf>;
```

Comment:
We cant' put it in `CSSLayoutStyleProps` because that is used only be the Grid/Flex _container_ styles, not the Grid/Flex item styles. We could create a new `CSSLayoutItemStyleProps` prop but it would only have a single property in it. I think we should leave it here.

Reply:
Oh yea